external/include/mbedtls/config.h

```diff
@@ -2866,7 +2866,9 @@
 #define MBEDTLS_LIGHT_DEVICE
 #if defined(MBEDTLS_LIGHT_DEVICE)
 
+#if !defined(CONFIG_STDK_IOT_CORE)
```

Comment:
In my opinion, STDK related configuration should be re-written in another file, such as config_stdk.h, and this header can be mapped into MBEDTLS_CONFIG_FILE.

Reply:
I will refine this config without changing this file, and this file modification will be removed, thank you